src/index.js

```diff
@@ -39,8 +39,8 @@ export { default as TimePicker } from './modules/timePicker.js';
 
 // utils
 export { default as DatePickerUtils } from './utils/datePickerUtils.js';
-export { default as DateUtils } from './utils/dateUtils.js';
-export { default as DOMUtils } from './utils/domUtils.js';
+export { default as dateUtils } from './utils/dateUtils.js';
```

Comment:
Gonna be a find/replace for the imports in HC I imagine?

Reply:
Yeah. I thought about not doing this change just yet, but I don't believe it'll be that time consuming go forth.